src/java.management.rmi/share/classes/javax/management/remote/rmi/RMIConnector.java

```diff
@@ -185,7 +185,7 @@ private RMIConnector(RMIServer rmiServer, JMXServiceURL address,
      * syntaxes, for example:</p>
      *
      * <pre>
-     * service:jmx:iiop://<em>[host[:port]]</em>/stub/<em>encoded-stub</em>
+     * service:jmx:protocolname://<em>[host[:port]]</em>/stub/<em>encoded-stub</em>
```

Comment:
"protocolname" could be read as literal here. I think change it back to "iiop" or replace it with something that like "myprotocol" that is more obviously an example protocol name.

Reply:
ok

Reply:
Could it really?  But ok, done. 8-)